Jenkinsfile

```diff
@@ -91,11 +132,23 @@ def RunIntegrationTest(k, v) {
                           export KUBECONFIG=$HOME/admin.conf
                           tests/scripts/helm.sh up'''
                     try{
+                        if ("${env.smokeOnly}" == "true") {
```

Comment:
is there any way to avoid the duplication in the script? For example, if true could you set a variable to `-test.run SmokeSuite`. If false the variable would be empty. Then use the variable in the bash script